plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -294,6 +314,17 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
       );
       callback();
     }
+    if (this._metricForceFlusher) {
+      this._metricForceFlusher().then(
+        () => callback(),
```

Comment:
Would you mind adding a test to verify that this callback is only invoked once when the tracer and meter are both registered?

Reply:
good catch!  I've added a test and it's failing.  Working on a fix now.

```
1) force flush
       should callback once after force flush providers:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:

2 !== 1

      + expected - actual

      -2
      +1
      
      at Context.<anonymous> (test/integrations/lambda-handler.force-flush.test.ts:236:12)
```

Reply:
Good catch, thanks for following up on this!

Reply:
https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1384 should fix this.